data/bus-times/express.yaml

```diff
@@ -1,27 +1,26 @@
-      - ['4:55pm',  '5:02pm',  '5:03pm',   '5:13pm',   '5:17pm',   '5:23pm',   '5:24pm',  '5:32pm']
-      - ['5:35pm',  '5:42pm',  '5:43pm',   '5:53pm',   '5:57pm',   '6:03pm',   '6:04pm',  '6:12pm']
-      - ['6:15pm',  '6:22pm',  '6:23pm',   '6:33pm',   '6:37pm',   '6:43pm',   '6:44pm',  '6:52pm']
-      - ['6:55pm',  '7:02pm',  '7:03pm',   '7:13pm',   '7:17pm',   '7:23pm',   '7:24pm',  '7:32pm']
-      - ['7:35pm',  '7:42pm',  '7:43pm',   '7:53pm',   '7:57pm',   '8:03pm',   '8:04pm',  '8:12pm']
-      - ['8:15pm',  '8:22pm',  '8:23pm',   '8:33pm',   '8:37pm',   '8:43pm',   '8:44pm',  '8:52pm']
-      - ['8:55pm',  '9:02pm',  '9:03pm',   '9:13pm',   '9:17pm',   '9:23pm',   '9:24pm',  '9:32pm']
-      - ['9:35pm',  '9:42pm',  '9:43pm',   '9:53pm',   '9:57pm',   '10:03pm',  '10:04pm', '10:12pm']
-      - ['10:15pm', '10:22pm', '10:23pm',  '10:33pm',  '10:37pm',  '10:43pm',  '10:44pm', '10:52pm']
+      - ['4:15pm',  '4:23pm',    '4:27pm',      '4:32pm',   '4:35pm',   '4:37pm',    '4:39pm',   '4:42pm',   '4:45pm',  '4:48pm',    '4:55pm']
+      - ['5:00pm',  '5:08pm',    '5:12pm',      '5:17pm',   '5:20pm',   '5:22pm',    '5:24pm',   '5:27pm',   '5:30pm',  '5:33pm',    '5:40pm']
+      - ['5:45pm',  '5:53pm',    '5:57pm',      '6:02pm',   '6:05pm',   '6:07pm',    '6:06pm',   '6:12pm',   '6:15pm',  '6:18pm',    '6:25pm']
+      - ['6:30pm',  '6:38pm',    '6:42pm',      '6:47pm',   '6:50pm',   '6:52pm',    '6:54pm',   '6:57pm',   '7:00pm',  '7:03pm',    '7:10pm']
+      - ['7:15pm',  '7:23pm',    '7:27pm',      '7:32pm',   '7:35pm',   '7:37pm',    '7:39pm',   '7:42pm',   '7:45pm',  '7:48pm',    '7:55pm']
+      - ['8:00pm',  '8:08pm',    '8:12pm',      '8:17pm',   '8:20pm',   '8:22pm',    '8:24pm',   '8:27pm',   '8:30pm',  '8:33pm',    '8:40pm']
+      - ['8:45pm',  '8:53pm',    '8:57pm',      '9:02pm',   '9:05pm',   '9:07pm',    '9:09pm',   '9:12pm',   '9:15pm',  '9:18pm',    '9:25pm']
+      - ['9:30pm',  '9:38pm',    '9:42pm',      '9:47pm',   '9:50pm',   '9:52pm',    '9:54pm',   '9:57pm',  '10:00pm', '10:03pm',   '10:10pm']
+      - ['10:15pm','10:23pm',   '10:27pm',     '10:32pm',  '10:35pm',  '10:37pm',   '10:39pm',  '10:42pm',  '10:45pm', '10:48pm',   '10:55pm']
 
   - days: [Su]
-    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454498, -93.161754], [44.433355, -93.186826], [44.448913, -93.171491], [44.454498, -93.161754], [44.460719, -93.155794], [44.462210, -93.183026]]
+    coordinates: [[44.462210, -93.183026], [44.460719, -93.155794], [44.454374, -93.162189], [44.438107, -93.182720], [44.432471, -93.187155], [44.435250, -93.188399], [44.441209, -93.181852], [44.449219, -93.171813], [44.454374, -93.162189], [44.460719, -93.155794], [44.462210, -93.183026]]
     stops:
-        [St. Olaf,  Carleton,  Food Co-op, Cub/Target, El Tequila, Food Co-op, Carleton,  St. Olaf]
+        [St. Olaf,  Carleton,  Food Co-op, Papa Murpheys, Cub/Target,      Arbys, Jesse James, El Tequila, Food Co-op,  Carleton,    St. Olaf]
```

Comment:
Ditto here—change `Papa Murpheys` to `Papa Murphys`? `Arbys` to `Arby's`? `Jesse James` to `Jesse James Lanes`?